tests/.infra/crw-ci/load-tests/Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+
+        string(name: 'numberOfUsers',
+                defaultValue: '',
+                description: 'Number of users for load tests.')
+
+        string(name: 'numberOfRepetitions',
+                defaultValue: '',
+                description: 'Number of workspaces that should be run by one user (in serial maneur).')
+
+        string(name: 'testingImage',
+                defaultValue: '',
+                description: 'Image with tests.')
+
+        string(name: 'repoWithScripts',
+                defaultValue: 'https://github.com/eclipse/che.git',
+                description: 'Github repo with load tests scripts.')
+        
+        string(name: 'pathToScripts',
+                defaultValue: 'che/tests/performance',
+                description: 'Path to scripts within the repo.')
+        
+        string(name: 'logLevel',
+                defaultValue: 'TRACE',
+                description: 'Log level for tests running in image.')
+
+    }
+    environment {
+        OC_PATH="/tmp"
+        PATH= "$PATH:$OC_PATH"
+
+        shmSize = "4096m"
+        loadPageTimeout = 240000
+        startWorkspaceTimeout = 900000
+        workspaceStatusPolling = 20000
+
+    }
+
+    stages {
+        stage('Preset environment') {
+            steps {
+                sh """
+                    set +x
+                    echo "Installing oc..."
+                    wget -q https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.3.1/openshift-client-linux-4.3.1.tar.gz -O - | tar -xz
+                    mv oc $OC_PATH
+                    echo "Setting CHE context..."
+                    oc login $cheClusterUrl -u $cheClusterUsername -p $cheClusterPassword --insecure-skip-tls-verify
+                    oc project $cheClusterNamespace
+                """
+                script {
+                    env.CHE_CONTEXT = sh (script: "oc whoami -c", returnStdout: true)
+                }
+                sh """
+                    set +x
+                    echo "Setting test context..."
+                    oc login $testClusterUrl -u $testClusterUsername -p $testClusterPassword --insecure-skip-tls-verify
+                    set +e
+                    project=`oc get projects | grep $testClusterNamespace`
+                    if [[ -z \$project ]]; then oc new-project $testClusterNamespace; fi
+                    set -e
+                    oc project $testClusterNamespace;
+                """
+                script {
+                    env.TEST_CONTEXT = sh (script: "oc whoami -c", returnStdout: true)
+                }
+            }
+        }
+        stage("Clone upstream che") {
+            steps {
+                sh"""
+                  echo "Clone che"
+                  git clone $repoWithScripts
+                """
+            }
+        }
+        stage('Run load tests') {
+            steps {
+                sh """
+                  ROOT=`pwd`
```

Comment:
Actually, it's not simple to read to code ten rows long.
I would suggest to divide logical blocks by empty rows to improve readability.